docs/index.md

```diff
@@ -339,6 +351,13 @@ resource "axual_application_principal" "log_scraper_in_production_principal" {
   principal = file("certs/certificate.pem")
 }
 
+resource "axual_application_principal" "connector_axual_application_principal" {
```

Comment:
Can we add some documentation here to explain to users that they need to decide how to securely store their secrets (certs, PK) before they use this functionality? We could add a link to this [blog](https://blog.gitguardian.com/how-to-handle-secrets-in-terraform/) for instance. @xsreality WDYT?

Reply:
Added security warning about private key in several places: https://github.com/Axual/terraform-provider-axual/pull/34/commits/5f9dc7e6ede83b7b6175a8627b656249b721d9fa

We also have a planned task for making a demonstration guide how to provide private key inside this Terraform provider's docs.